client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.service.js

```diff
@@ -0,0 +1,13 @@
+angular.module('services').service('cdnDedicatedManageLogs', function (OvhHttp) {
```

Comment:
This is a deprecated way of doing things, please use https://github.com/ovh-ux/ovh-api-services instead :)